### PR TITLE
vendor: remove unused libraries

### PR DESCRIPTION
**Pull request by nonsense**
This PR is:
1. Removing unused libraries from vendor.
2. Re-enabling tests that are not Swarm-specific - hopefully they've been fixed by other teams by now.

### CODE REVIEW

**Review by janos — APPROVED** [review]
LGTM, just `govendor list` reports that `github.com/influxdata/influxdb/client/v2` is unused too.

**Comment by holisticode** [comment]
Shouldn't `govendor list` be included in the test suite somehow so there's always a report if a library becomes obsolete?

File: p2p/nat/natupnp_test.go [hunk 1]
```diff
@@ -29,7 +29,6 @@ import (
 )
 
 func TestUPNP_DDWRT(t *testing.T) {
-	t.Skip("broken")
```
Comment (holisticode) [inline p2p/nat/natupnp_test.go:32]: This change is not related to removing vendoring....what's the reason to re-enable this test?

File: whisper/whisperv6/peer_test.go [hunk 2]
```diff
@@ -101,7 +101,7 @@ var debugMode = false
 var prevTime time.Time
 var cntPrev int
 
-func XTestSimulation(t *testing.T) {
+func TestSimulation(t *testing.T) {
```
Comment (holisticode) [inline whisper/whisperv6/peer_test.go:104]: Same here, rationale for re-enabling this test?

**Review by holisticode — APPROVED** [review]

File: p2p/nat/natupnp_test.go (hunk 1 above)
Comment (nonsense) [reply p2p/nat/natupnp_test.go:32]: See comment on PR - these tests are not related to Swarm, and I don't see them failing on go-ethereum master.

File: whisper/whisperv6/peer_test.go (hunk 2 above)
Comment (nonsense) [reply whisper/whisperv6/peer_test.go:104]: See comment on PR - these tests are not related to Swarm, and I don't see them failing on go-ethereum master.
